.github/workflows/release.yml

```diff
@@ -1,18 +1,15 @@
-
```

Comment:
I think this file is still read only which makes GitHub think it's a generated file. Might want to change that?

Reply:
it was the `.gitattributes` file that projen generated. I removed it and it seems to have fixed the issue.